src/Finalizer/Console/CheckFinalClassesCommand.php

```diff
@@ -43,25 +44,30 @@ public function __construct()
     /**
      * {@inheritDoc}
      */
-    public function execute(InputInterface $input, OutputInterface $output)
+    public function execute(InputInterface $input, OutputInterface $output) : int
     {
         $classScanner = new DirectoryClassScanner();
         $fileScanner  = new DirectoryFileScanner();
         $classes      = $classScanner($fileScanner($input->getArgument('directories')));
 
-        $this->renderClassesToFinalize($classes, $output);
-        $this->renderClassesToDeFinalize($classes, $output);
+        $finalizeStatusCode = $this->renderClassesToFinalize($classes, $output);
+        $deFinalizeStatusCode = $this->renderClassesToDeFinalize($classes, $output);
+
+        return max($finalizeStatusCode, $deFinalizeStatusCode);
     }
 
     /**
      * @param \ReflectionClass[] $classes
      * @param OutputInterface    $output
      */
-    private function renderClassesToFinalize(array $classes, OutputInterface $output)
+    private function renderClassesToFinalize(array $classes, OutputInterface $output) : int
     {
+        $statusCode = 0;
         $tableHelper = new Table($output);
 
-        if ($toFinalize = $this->getClassesToFinalize($classes)) {
+        $toFinalize = $this->getClassesToFinalize($classes);
+        if (count($toFinalize) > 0) {
+            $statusCode = 1;
```

Comment:
Instead of using a variable, use `return 1` in this `if`, and `return 0` at the end